Add benchmarks

torralbaa opened the pull request:
Added some benchmarks. While these are pretty basic, the results should convince the clients.

Closes #5.

torralbaa committed:
Add benchmarks

torralbaa committed:
Fix typo

qntm reviewed (changes requested):
Thank you for your contribution! It looks like this PR isn't quite ready for merging yet.

* I don't think we need both `benchmarks.js` and `src/benchmarks.js` with seemingly identical content? For preference I would put the benchmarking code in its own directory.
* This repo doesn't use `package-lock.json`, so please remove that from the PR for now. I believe this is causing CI failures, also. I won't merge a PR with failing CI.
* This repo uses Standard JS. So, please make sure to run Standard against your new code and fix all of the flagged issues. Don't use semicolons, use `const` instead of `var`, and so on. (Apologies that linting doesn't run automatically as part of `npm run test`, I may fix that myself separately.)
* An `npm run` script for running the benchmarks would be appreciated.

torralbaa committed:
Made requested changes.

* Removed package-lock.json
* Added npm run script for benchmarks
* Fixed issues detected by Standard JS
* Deleted duplicated file

torralbaa commented:
Hello,

> I don't think we need both benchmarks.js and src/benchmarks.js with seemingly identical content?

This was an error, sorry.

Fixed all the other issues and added `package-lock.json` to the `.gitignore` file for future contributors.

qntm reviewed (approved):
```
> node ./benchmarks/index.js

Running benchmarks...
fastjson.stringify x 911,449,851 ops/sec ±1.03% (89 runs sampled)
JSON.stringify x 60.50 ops/sec ±0.71% (59 runs sampled)
Fastest is: fastjson.stringify
fastjson.parse x 53,947,644 ops/sec ±1.90% (82 runs sampled)
JSON.parse x 46.03 ops/sec ±1.01% (56 runs sampled)
Fastest is: fastjson.parse
```

Astonishing stuff.
